lib/active_merchant/billing/gateways/cyber_source.rb

```diff
@@ -1079,6 +1080,10 @@ def message_from(response)
           response[:message]
         end
       end
+
+      def eligible_for_zero_auth?(payment_method, options = {})
+        payment_method.is_a?(CreditCard) && options[:zero_amount_auth]
```

Comment:
Apologies for the switcheroo, but I just remembered an issue we've had recently with a similar field name being confusing for users, since this flag is specifically used for `verify` rather than `authorize`, and the suggestion was to keep the name as specific as possible.
```suggestion
        payment_method.is_a?(CreditCard) && options[:zero_amount_verify]
```

Reply:
But now worries - I can make this change as part of the merge.